client/client.go

```diff
@@ -290,21 +291,30 @@ func NewClientWithContext(ctx context.Context, svrAddrs []string, security Secur
 	c, clientCtx, clientCancel, tlsCfg := createClient(ctx, &security)
 	c.tsoStreamBuilderFactory = &pdTSOStreamBuilderFactory{}
 	c.bc = newPDBaseClient(clientCtx, clientCancel, &c.wg, addrsToUrls(svrAddrs), tlsCfg, c.option)
+	c.tsobc = c.bc
```

Comment:
the same two clients?

Reply:
tsobc is an interface of tso baseclient which can be either tso in pd client or tso microservice client. For the former, tso in pd client is the pd client itself, that's why c.tsobc = c.bc. Actually, I planned one more refactor to make it much clearer -- Client contains pdServiceDiscovery by renaming pdBaseClient after we move all the tso related logic out of pdBaseClient. c.tsobc will be backed by either pdServiceDiscovery or tsoServiceDiscovery. After that, we won't see 'c.tsobc = c.bc' such thing which looks weird at the first sight.